server/application.py

```diff
@@ -52,28 +57,18 @@ def routes(route_ids_string):
     return flask.Response(json.dumps(route_data), mimetype="application/json")
 
 
-@application.route("/statistics")
-def statistics():
-    summaries_by_route = get_cached_summaries()
-    return flask.Response(json.dumps(summaries_by_route), mimetype="application/json")
-
-
 # root function to serve landing page
-# add ?testMode=1 to enable test mode
-#   if test mode is true, app will show all trains, not just new ones
-# if shell mode is true, app will generate without getting any intital API data
 @application.route("/")
 def root():
-    test_mode = flask.request.args.get("testMode")
     shell = flask.request.args.get("shell")
     if shell:
         return static_files("index.html")
-    initial_data = asyncio.run(
-        mbta_api.initial_request_data(DEFAULT_ROUTE_IDS, test_mode)
+    payload = asyncio.run(
```

Comment:
Nothing has changed about this. Just changed the variable name so `initial_data` could be a module name.